### DISCUSSION TITLE
Changelog after Version 5.0

### DISCUSSION THREAD

**Discussion by araccaine** [PaulWoitaschek/Voice#1724] [General]
Hi folks, 
is there any changelog for newer Voice versions available?
F-Droid links to https://github.com/PaulWoitaschek/Voice/blob/HEAD/CHANGELOG.md but unfortunately there are no changes after Version 5 listed.

PS:
Thanks a ton for this app :)

**Comment by srphillips** [comment]
Seems like the maintainer has removed the CHANGELOG.md file in-favor of enabling the generated release notes, but that only works from the version when it was enabled forward.  Here's the recent commit with the change: https://github.com/PaulWoitaschek/Voice/commit/ff8d8308aca8cfc99a3d93abaa15e32d803177e3
